libs/@local/hash-graph-client/api.ts

```diff
@@ -492,25 +492,6 @@ export interface Entity {
    */
   properties: object;
 }
-/**
- *
- * @export
- * @interface EntityEditionId
- */
-export interface EntityEditionId {
```

Comment:
Seems like the REST API ended up changed anyway. I appreciate the shape of the objects should probably still be the same, but were any of these types imported?